css/michael-le.css

```diff
@@ -0,0 +1,16 @@
+h1 {
+  color: red;
+}
+#green {
+  color: green;
+}
+.dates {
+  font-weight: bold;
+  font-style: italic;
+}
+ul > li {
```

Comment:
Check out the updated task for using a `child selector` in this assignment, and update this CSS rule.